.github/pull_request_template.md

```diff
@@ -0,0 +1,16 @@
+<!--
+Thanks for filing a pull request!
+Reviewers are typically assigned within a week of filing a request.
+To learn more about code review, see our documentation on Tree Hygiene: https://github.com/flutter/flutter/blob/main/docs/contributing/Tree-hygiene.md
+-->
+
+*Replace this description with a description of what your PR is changing, adding, enabling, or disabling.*
+
+> [!IMPORTANT]
+> This repository is a testing suite that contains references to tests (in the `registry` directory) that are run with every commit to Flutter
+> to verify that no breaking changes have been introduced (in the "customer_testing" shards). Your merged PR is _not_ automatically run in the
+> Flutter CI tree.
+>
+> Once merged, a PR must be sent updating `dev/customer_testing/tests.version` with the latest SHA.
```

Comment:
```suggestion
> After merging this PR, you must send another PR to flutter/flutter updating `dev/customer_testing/tests.version` in that repo with the latest git commit SHA of the flutter/test repo.
```

Reply:
Would this not be solved by moving it all to `flutter/flutter/customer_testing`?  🛑 🗡️🦆☠️

Reply:
Yes, in some hypothetical future it would.